keyboards/opticalkb/rev1/config.h

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 Girish Palya (@girishji)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEFD
+#define PRODUCT_ID 0x5302
+#define DEVICE_VER 0x0002
+#define MANUFACTURER girish
+#define PRODUCT opticalkb
```

Comment:
This needs to be migrated to the info.json file.

Reply:
Please review, I committed the requested changes.